pyscriptjs/src/components/base.ts

```diff
@@ -124,8 +124,9 @@ export class BaseEvalElement extends HTMLElement {
                                  : this.getSourceFromElement();
             this._register_esm(runtime);
 
-            <string>await runtime.run(`set_current_display_target(element="${this.id}")`);
+            <string>await runtime.run(`set_current_display_target(target_id="${this.id}")`);
             <string>await runtime.run(source);
+            <string>await runtime.run(`set_current_display_target(target_id=None)`);
```

Comment:
Thanks! I completely forgot about it.
This is what you mean:
```
            try {
                <string>await runtime.run(`set_current_display_target(target_id="${this.id}")`);
                <string>await runtime.run(source);
            } finally {
                <string>await runtime.run(`set_current_display_target(target_id=None)`);
            }
```
correct?

Reply:
yes. And ideally, with a test ;)
The test is a bit hard to write, but I I guess that with the current code the following test would fail:
1. execute a `<py-script>` which raises an exception
2. call display() from an event handler: in theory it should fail (because you cannot call display() without explicit target form event handlers), but with the current code it will pass because the current_target was set in (1) and never reset to None.

So the test should check that (2) fails.